docs/basic-docs/atomicdex-api-20-dev/withdraw_tasks.md

```diff
@@ -0,0 +1,360 @@
+# Withdraw Tasks
+
+
+
+## withdraw\_init
+
+The `task::withdraw::init` method generates, signs, and returns a transaction that transfers the `amount` of `coin` to the address indicated in the `to` argument. The status of this method can be queried via the [withdraw_status](#withdraw-status) method, or cancelled with [withdraw_cancel](#withdraw-cancel).
+
+:::tip
+When used for ZHTLC coins like ARRR or ZOMBIE, it may take some time to complete.
+:::
+
+
+### Arguments
+
+| Structure     | Type             | Description                                                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| coin          | string           | The name of the coin the user desires to withdraw                                                                                         |
+| to            | string           | Coins are withdrawn to this address                                                                                                       |
+| amount        | string (numeric) | The amount the user desires to withdraw, ignored when `max=true`                                                                          |
+| memo          | string           | Optional, used for ZHTLC and Tendermint coins only. Attaches a memo to the transaction.                                                   |
```

Comment:
Please, add @artemii235 note: We advise not to use it to withdraw to CEXes requiring memo.